src/envoy/http/authn/authenticator_base.cc

```diff
@@ -0,0 +1,101 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/envoy/http/authn/authenticator_base.h"
+#include "src/envoy/http/authn/filter_context.h"
+#include "src/envoy/http/authn/mtls_authentication.h"
+#include "src/envoy/utils/utils.h"
+
+namespace iaapi = istio::authentication::v1alpha1;
+
+namespace Envoy {
+namespace Http {
+namespace IstioAuthN {
```

Comment:
How about separate it as

namespace Istio
namespace AuthN

Reply:
Done